exporter/stats/stackdriver/stackdriver.go

```diff
@@ -242,6 +242,10 @@ func (e *Exporter) createMeasure(ctx context.Context, vd *stats.ViewData) error
 	switch agg.(type) {
 	case stats.CountAggregation:
 		valueType = metricpb.MetricDescriptor_INT64
+	case stats.SumAggregation:
```

Comment:
This can't work without updating equalAggWindowTagKeys.

Reply:
Thanks for catching this, I updated this in https://github.com/census-instrumentation/opencensus-go/pull/224